Source/DataSources/ParticleSystemGraphics.js

```diff
@@ -0,0 +1,331 @@
+     * @constructor
+     *
+     * @param {Object} [options] Object with the following properties:
+     */
+    function ParticleSystemGraphics(options) {
+        this._show = undefined;
+        this._showSubscription = undefined;
+        this._image = undefined;
+        this._imageSubscription = undefined;
+        this._startScale = undefined;
+        this._startScaleSubscription = undefined;
+        this._endScale = undefined;
+        this._endScaleSubscription = undefined;
+        this._startColor = undefined;
+        this._startColorSubscription = undefined;
+        this._endColor = undefined;
+        this._endColorSubscription = undefined;
+        this._rate = undefined;
+        this._rateSubscription = undefined;
+        this._minWidth = undefined;
+        this._minWidthSubscription = undefined;
+        this._maxWidth = undefined;
+        this._maxWidthSubscription = undefined;
+        this._minHeight = undefined;
+        this._minHeightSubscription = undefined;
+        this._maxHeight = undefined;
+        this._maxHeightSubscription = undefined;
+        this._minSpeed = undefined;
+        this._minSpeedSubscription = undefined;
+        this._maxSpeed = undefined;
+        this._maxSpeedSubscription = undefined;
+        this._minLife = undefined;
+        this._minLifeSubscription = undefined;
+        this._maxLife = undefined;
+        this._maxLifeSubscription = undefined;
+        this._loop = undefined;
+        this._loopSubscription = undefined;
+        this._lifeTime = undefined;
+        this._lifeTimeSubscription = undefined;
+        this._emitterModelMatrix = undefined;
+        this._emitterModelMatrixSubscription = undefined;
+        this._bursts = undefined;
+        this._burstsSubscription = undefined;
+        this._forces = undefined;
+        this.__forcesSubscription = undefined;
+
+        this._definitionChanged = new Event();
+
+        this.merge(defaultValue(options, defaultValue.EMPTY_OBJECT));
+    }
+
+    defineProperties(ParticleSystemGraphics.prototype, {
+        /**
+         * Gets the event that is raised whenever a property or sub-property is changed or modified.
+         * @memberof ParticleSystemGraphics.prototype
+         * @type {Event}
+         * @readonly
+         */
+        definitionChanged : {
+            get : function() {
+                return this._definitionChanged;
+            }
+        },
+
+        /**
+         * Gets or sets the boolean Property specifying the visibility of the model.
+         * @memberof ParticleSystemGraphics.prototype
+         * @type {Property}
+         * @default true
+         */
+        show : createPropertyDescriptor('show'),
+
+
+        /**
+         * The particle emitter for this particle system.
+         * @memberof ParticleSystemGraphics.prototype
+         * @type {Property}
+         * @default new CircleEmitter({radius: 0.5});
+         */
+        emitter : createPropertyDescriptor('emitter'),
```

Comment:
For now, yes, they would just create multiple particle systems.  I played around with ParticleSystems having multiple emitters, and it worked just fine.  It was hard to really distinguish what would be the difference between multiple emitters and multiple systems, so I just tried to keep it simple for now.